### PR TITLE
[BPK-2236] Fix range render issue in calendar

### PR DESCRIPTION
**Pull request by k0nserv**
Fix an issue were when the calendar initialy rendered with a range
selected the component would render in a broken state.

### CODE REVIEW

File: packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m [hunk 1]
```diff
@@ -45,6 +50,40 @@ - (UIView *)view {
...
+RCT_EXPORT_METHOD(forceRender:(nonnull NSNumber *)reactTag) {
+    [self.bridge.uiManager addUIBlock:
+     ^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry){
+         UIView *view = viewRegistry[reactTag];
+
+         if ([view isKindOfClass:[BPKCalendar class]]) {
+             BPKCalendar *calendar = (BPKCalendar *)view;
+             NSArray<NSDate* > *selectedDates = calendar.selectedDates;
+
+             calendar.selectedDates = @[];
```
Comment (karanjthakkar) [inline packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m]: Do we need to empty the calendar once for it to render correctly on the second reload? Is this a temporary fix or is this going to be around?

Comment (k0nserv) [reply packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m]: It seems to be required yeah

**Comment by k0nserv** [comment]
> LGTM 👍 Might be a good idea though to figure out the underlying cause for this hack to be required.

Yeah that's fair, on native iOS if you create the calendar in `viewDidLoad` and then configure it in`viewDidAppear` it doesn't exhibit this behaviour. However if you configure it in `viewDidLoad` the same thing happens. @Szabadember might know more about the potential root cause

File: UNRELEASED.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # Unreleased
 
 _Nothing yet..._
```
Comment (georgegillams) [inline UNRELEASED.md]: Can we remove this line (and never add it again)?

Comment (k0nserv) [reply UNRELEASED.md]: I'll clean it up when I release

**Pull request merged**